client/src/components/PreviousTripsCard/PreviousTripsCard.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import "./PreviousTripsCard.css";
+import Card from "../Card/Card";
+
+// Function based React Component
+const PreviousTripsCard = props => {
+  let { link, trips } = props;
+  let [title, tripsItemOne, tripsItemTwo] = [
+    props.children[0],
+    props.children[1],
+    props.children[2]
+  ];
+
+  return (
+    <Card color="purple">
+      <div className="PreviousTripCard__container">{title}</div>
+      {trips ? (
+        <div className="PreviousTripCard__trips">
+          {tripsItemOne}
+          <span className="PreviousTripCard__trip-two">{tripsItemTwo}</span>
+          <a href={link} className="PreviousTripCard__arrow" />
```

Comment:
Can you make this arrow a little bit smaller and add some padding to it? It looks a little bit crowded against the edge of the TripCard and also the Card container.